book/basics/data_wrangling.nim

```diff
@@ -41,6 +41,7 @@ nbText: """
 Or if one already has a mix of sequences and tensors of the same length:
 """
 nbCode:
+  import datamancer
```

Comment:
I think like this you'll run into "duplicate import of `datamancer`" in the CI, because the generated Nim file that is compiled now has 2 `import` because of the above.

We should probably just add a first block that only imports the two modules with a text like "let's import all the modules we are going to need here" sort of thing.